-y option for install_deps.sh script to allow default accept Installation (#1094)
huawei-noah/SMARTS: README.md

> # Follow the instructions given by prompt for setting up the SUMO_HOME environment variable
> bash utils/setup/install_deps.sh
> 
> # Alternatively you can use the -y option to allow the default accept of installation to prevent timeout from waiting for user input. 
> # Though you still need to follow the SUMO_HOME environment variable instructions after the script finishes executing.
> bash utils/setup/install_deps.sh -y
> 

Reviewer: I think we can condense the instructions, this might be enough: 
```suggestion
# Install the system requirements then follow the instructions given by prompt for setting up the SUMO_HOME environment variable
# You can use the -y option to allow the default accept of installation to prevent timeout from waiting for user input. 
bash utils/setup/install_deps.sh
```
